images/vct/Dockerfile

```diff
@@ -31,10 +31,10 @@ RUN echo ${TARGETARCH}
 
 RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -o build/bin/vct cmd/vct/main.go
 
-ARG ALPINE_VER
+FROM scratch
 
-FROM alpine:${ALPINE_VER} as base
 LABEL org.opencontainers.image.source https://github.com/trustbloc/vct
-RUN apk add -U --no-cache ca-certificates
+
```

Comment:
And the next step is to copy needed files from an alpine multi-stage builder (e.g., ca-certificates).